js/controllers/SiteHandler.js

```diff
@@ -40,7 +40,9 @@
 		this.bind('notifyUser', this.fetchNotificationHandler_);
 		this.bind('reloadTab', this.reloadTabHandler_);
 		this.bind('callWhenClickOutside', this.callWhenClickOutsideHandler_);
-		this.bind('mousedown', this.mouseDownHandler_);
+		$('body').on('mousedown', function (e) {
```

Comment:
The previous `bind(...)` actually swallows the event for itself (there's a `e.stopPropagation()`), but I didn't see a reason for it in this case, so I just had it replaced...

We need the `mouseup` event to bubble up in order to reach other generic handlers.